modules/CmpdReg/src/server/routes/CmpdRegRoutes.coffee

```diff
@@ -335,14 +332,38 @@ exports.getLotDependenciesInternal = (lot, user, allowedProjects, includeLinkedL
 			# This way it'know there are experiments linked that aren't readable
 			if !acls.getRead()
 				console.log "Experiment #{experiment.experiment.codeName} is not readable by user #{user.username}"
-				# If the experiiment is not readable we just want to include the acls but not the experiment code table
+				# If the experiment is not readable we just want to include the acls but not the experiment code table
 				# We include the acls so that the user can see that there is an experiment linked that they can't read
 				dependencies.linkedExperiments[idx] = {acls: acls, code: null, name: null, ignored: false}
 			else
-				# The experiment is readable so includ the experiment and acls
-				dependencies.linkedExperiments[idx].acls = acls
+				# The experiment is readable so include the experiment and acls
+				linkedExperiment = dependencies.linkedExperiments[idx]
+				linkedExperiment.acls = acls
+		
+		# Add protocol and analysis group codes to user accessible experiments
+		for linkedExperiment in dependencies.linkedExperiments
+			if not linkedExperiment.acls.getRead()
+				continue
+			experiment = _.findWhere(experiments, {experimentCodeName: linkedExperiment.code})
+			linkedExperiment.protocolCode = experiment.experiment.protocol.codeName
+			analysisGroupCodes = _.pluck(experiment.experiment.analysisGroups, "codeName")
+			linkedExperiment.analysisGroupCodes = analysisGroupCodes.sort()
```

Comment:
I think the [original ticket](https://jira.schrodinger.com/browse/ACAS-307) referenced digging down into the analysis groups -> states -> values and plucking the values like:

```
result type
values 
```

I would think the users would benefit from fetching the analysis group values from unignored analysis group, states and values e.g output:
```
analysis_group_results: [
{ 
    "id": 1
    "lsType": "stringValue"
    "lsKind": "My String Answer"
    "stringValue": "hello"
}, {
{ 
    "id": 2
    "lsType": "numericValue"
    "lsKind": "My Numeric Answer"
    "numericValue": 1234.1234
},
```

Reply:
@bffrost Does that seem right to you?

We'll probably additionally want to filter out all analysis group values where lsKind = "batch code" as that is where the lot corp name is which is already implied by the output of this data.

Reply:
Sorry for the delayed reply, but yes what @brianbolt says is right. @iamanandkr it'd be good to include the analysis group values in the way that Brian showed

Reply:
Done.